ktlint-test/src/main/kotlin/com/pinterest/ktlint/test/RuleSetProviderTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.pinterest.ktlint.test
+
+import com.pinterest.ktlint.core.RuleSetProvider
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import java.io.File
+
+open class RuleSetProviderTest(
```

Comment:
I'm not sure that this is the best way for the centralized test code - this could be just a simple top-level function, which may be called from within a test method (e.g. in case `StandardRuleSetProviderTest` contains other tests), so I'm open for changing this, but currently it works.